helm/rpm-builder/templates/flotta-rpm-build-task.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  namespace: {{ .Values.targetNamespace }}  
+  name: flotta-build-task
+spec:
+  workspaces:
+    - name: workspace
+  params:
+    - name: source-dir
+      description: Directory Contaning source File
+      type: string
+  results:
+    - name: build-commit
+      description: Commit ID of the Build
+  steps:
+    - name: build-image-playbook
+      image: {{ .Values.buildTask.image.name }}:{{ .Values.buildTask.image.tag }}
+      workingDir: /workspace/workspace
+      args:
+        - |
+          ansible-galaxy collection install -r tools/ansible/collections/requirements.yaml
+          ansible-playbook \
+            -e source_dir=$(params.source-dir) \
```

Comment:
I might have missed it, but I didn't see where you are using `source_dir` or `build_commit_output_file`.

Reply:
`build_commit_output_file` is not necessary. I'll remove it. As for `source_dir` I will rename to `base_repo_dir` and make sure it's being used as the base repository for both jobs.